components/gitpod-cli/cmd/preview.go

```diff
@@ -32,6 +35,21 @@ var previewCmd = &cobra.Command{
 		url = replaceLocalhostInURL(service, url)
 
 		_, err = service.OpenPreview(theialib.OpenPreviewRequest{URL: url})
+		if err == theialib.ErrNotFound {
+			gpPreviewBrowser := os.Getenv("GP_PREVIEW_BROWSER")
+			if gpPreviewBrowser != "" {
+				gpPreviewArgs := strings.Fields(gpPreviewBrowser)
```

Comment:
@csweichel Does it look correct to support command with args? We probably should the same for `GP_OPEN_EDITOR` otherwise `exec.LookPath` will fail

Reply:
Indeed, we should do that for `GP_OPEN_EDITOR` as well.
I'd be happy to do this in a follow-up PR.